package.json

```diff
@@ -36,6 +36,9 @@
     "platformNames": [
       "Windows"
     ],
+    "scripts": {
+      "install-wad": "./scripts/install-wad.mjs"
+    },
     "mainClass": "WindowsDriver"
   },
   "files": [
```

Comment:
Need to add `scripts` and remove `install-npm.js`?

Reply:
good catch. I've almost  forgotten about it